sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventActionsPage.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.pages.application.events;
+
+import org.openqa.selenium.By;
+
+public class EventActionsPage {
+  public static final By CREATE_BUTTON = By.id("actionCreate");
+  public static final String EDIT_SPECIFIC_EVENT_BUTTON =
```

Comment:
It would be good to have "FIRST" instead of "SPECIFIC" as it points always to the first item in the list

Reply:
sure, I will change it tomorrow :D

Reply:
Perfect!!